### DISCUSSION TITLE
Facing issue with Content-Length header after upgrading to aiohttp 3.10.2 from aiohttp 3.1.2

### DISCUSSION THREAD

**Discussion by VictoriaTskhondia** [aio-libs/aiohttp#8737] [Q&A]
...

**Comment by Dreamsorcerer** [comment]
3.1.3? That's 6.5 years of changes...

Can you provide a full reproducer, your current one has an indentation error, missing imports, multiple variables which aren't defined...

**Comment by webknjaz** [comment]
I recommend upgrading in smaller increments. Bumping the version of aiohttp and CPython one after another so that you'd be able to see which version changed the behavior and inspect the corresponding change log entries.
